server/models/contests/addGroup.js

```diff
@@ -13,30 +13,17 @@ function addGroup({ params, body, username }) {
     const { contest_id: contestId } = params
     const { group_id: groupId } = body
     pool.query(
-      `SELECT * FROM contests_moderators WHERE contest_id=? AND moderator=?`,
-      [contestId, username],
-      (error, results) => {
-        if (error) {
+      `INSERT INTO contests_groups (contest_id, group_id) VALUES ((SELECT contest_id FROM contests_moderators WHERE contest_id=? AND moderator=?), ?)`,
```

Comment:
It is just `INSERT INTO ... SELECT`
`VALUES` shouldn't come here

Reply:
Use the exact `sql` which I gave

Reply:
that was throwing error, that's why i used values

Reply:
This should work..I gave this
```
 `INSERT INTO contests_groups (contest_id, group_id) SELECT contest_id, ? FROM contest_moderators WHERE contest_id=? AND moderator=?`,